ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContext.scala

```diff
@@ -39,16 +42,16 @@ private[kvutils] trait CommitContext {
 
   /** Set a value in the output state. */
   def set(key: DamlStateKey, value: DamlStateValue): Unit = {
-    if (!outputs.contains(key)) {
-      outputOrder += key
+    if (inputAlreadyContains(key, value)) {
+      logger.trace(s"Identical output found for key $key")
```

Comment:
If you use an Slf4j logger, you may want to use Slf4j templates instead of Scala string interpolation.

Reply:
Thanks, that's a good point especially in case of expensive `.toString` methods.